assets/preload.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Can just be a list of objects?

Reply:
Yup. Old habits worrying about JSON hijacking, will update.